exonum/src/node/mod.rs

```diff
@@ -30,13 +30,12 @@ use api::{
 };
 use failure;
 use futures::{sync::mpsc, Future, Sink};
-use tokio_core::reactor::Core;
-use toml::Value;
-
 use std::{
     collections::{BTreeMap, HashSet}, fmt, io, net::{SocketAddr, ToSocketAddrs}, sync::Arc, thread,
     time::{Duration, SystemTime},
 };
+use tokio_core::reactor::Core;
+use toml::Value;
```

Comment:
I think we should keep imports [in order](https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions):

> * Imports (use):
>   * Third-party libraries.
>   * Standard library.
>   * Internal.

Reply:
@ilammy as I know `rustfmt` does this job correctly.

Reply:
Apparently it doesn't do a good job for our code style. For example, [this place](https://github.com/exonum/exonum/blob/669ca87e941318e2b7f383485448f9f2ecab60e7/services/configuration/src/lib.rs#L81-L91) is total mess of imports and reexports that should be kept separate according to the code style.